Maquetado/views/index.html

```diff
@@ -10,25 +10,34 @@
 </head>
 
 <body>
-  <div class='bookContainer'>
-      <img src='../resources/img/book-cover.png' />
-    <div class='textContainer'>
-      <h1 class="mainTitle">
-        Título del libro
-        <strong class='bookGenre'> (género)</strong>
-      </h1>
-      <h2 class="title">
-        Autor del libro:
-        <strong class="bookDetail">Nombre del autor del libro</strong>
-      </h2>
-      <h2 class="title">
-        Editorial:
-        <strong class="bookDetail">Nombre de la editorial</strong>
-      </h2>
-      <h2 class="title">
-        Año de publicación:
-        <strong class="bookDetail">Año de publicación</strong>
-      </h2>
+
+  <div class='pageContainer'>
+    <div class='backButtonContainer'>
+      <a href='http://127.0.0.1:5500/Maquetado/views/bookList.html' class='backButton'> Atrás </a>
+      <a href='http://127.0.0.1:5500/Maquetado/views/bookList.html'>
+        <img src='../resources/img/back-arrow.png'class='backArrowImage' /> 
+      </a>
+    </div>
+    <div class='bookContainer'>
+      <img src='../resources/img/book-cover.png' class='bookImage' />
```

Comment:
`alt` attribute is missing.
the alt attribute is very important for the accessibility.